lib/internal/process/next_tick.js

```diff
@@ -281,7 +281,7 @@ function setupNextTick() {
     if (process._exiting)
       return;
 
-    if (triggerAsyncId === null) {
+    if (triggerAsyncId === null || triggerAsyncId === undefined) {
```

Comment:
I belive `!Number.isSafeInteger(triggerAsyncId ) || triggerAsyncId < 0` is the "preferred" test